charts/meetings/values.yaml

```diff
@@ -0,0 +1,80 @@
+# Default values for meetings.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: jimbersoftware/freeflowconnect_grid
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "0.7"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: NodePort
```

Comment:
NodePort service on 80? by default nodeports are on the range of 30k range How did you achieve that?

Reply:
As the old marketplace used to work, every solution got an FQDN. There was a reverse proxy to achieve that. In the mindset that everything will still have its own FQDN, I just put a reverse proxy in front myself to make the port forward to 80 locally.

Reply:
this is not the right way to do in kubernetes. in kubernetes your run your service in the cluster, and if you want to expose you have multiple ways
1- NodePort the easiest, but then it opens a port -random port- on a specific range based on the cluster config usually above 31000 on all nodes 
2- Using ingress which is kinda like your reverse proxy running on 80, 443 and that proxies the traffic to another service on the cluster (which is almost all cases)